packages/core/src/App/Containers/OneTimeDepositModal/one-time-deposit-modal.scss

```diff
@@ -0,0 +1,65 @@
+.one-time-deposit-modal {
+    display: flex;
+    align-self: stretch;
+
+    &__body {
+        display: flex;
+        justify-content: center;
+        height: 80vh;
+
+        @include mobile {
+            height: 100%;
+        }
+    }
+
+    &__content {
+        width: 60%;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+
+        @include mobile {
+            width: 100%;
+            padding: 2rem;
+        }
+    }
+
+    &__description {
+        display: grid;
+        grid-template-columns: 78fr 22fr;
+        gap: 0.5rem;
```

Comment:
is 0.5rem the right value as per figma design file?

Reply:
let me re-check it 🙏